aws-rds/README.md

```diff
@@ -28,99 +28,172 @@ To access this integration, [connect to CloudWatch](https://github.com/signalfx/
+* [Building from source](#building-from-source)
+* [Metrics collected by this integration](#metric-groups-collected-by-this-integration)
+
+##### Note: Encryption of your SignalFx access token
+This Lambda function uses your SignalFx access token to send metrics to SignalFx, as an environment variable to the function. While Lambda encrypts all environment variables at rest and decrypts them upon invocation, AWS recommends that all sensitive information such as access tokens be encrypted using a KMS key before function deployment, and decrypted at runtime within the code. 
+
+Both procedures below include instructions for using either an encrypted or non-encrypted access token. 
+
+#### Deploying through the Serverless Application Repository
+
+##### 1. Set up an encryption key and encrypt your access token (if desired)
+Only follow this step if you chose to manually encrypt your access token.
+Either create a new KMS encryption key or select a preexisting one. **The key
+must be in the same availability zone as the RDS instances you are
+monitoring.** You can create and manage encryption keys from IAM in the AWS
+management console. Documentation on KMS encryption from the CLI can be found
+[here](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html). Make sure you have access to the cipher text output by the encryption as well as the key id of the encryption key you used.
+
+##### 2. Create the Lambda function
+Click `Create Function` from the list of Lambda functions in your AWS console.
+Make sure you are in the intended availability zone. Select the
+`Serverless Application Repository` option in the upper right hand corner.
+Search for `signalfx rds` and choose the appropriate entry based on whether you
+encrypted your access token.
+
+To access the templates directly, find the template for encrypted access
+tokens [here](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:134183635603:applications~signalfx-enhanced-rds-metrics-encrypted). 
+The template for non-encrypted access tokens is [here](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:134183635603:applications~signalfx-enhanced-rds-metrics).
+
+##### 3. Fill out application parameters
+Under `Configure application parameters`, choose a name for your function,
+and fill out the fields accordingly.
+
+**Parameters for template using encrypted access tokens**
+- `EncryptedSignalFxAuthToken`: The Ciphertext blob output from your encryption of your SignalFx organization's access token
+- `KeyId`: The key id of your KMS encryption key; it is the last section of the key's ARN.
+- `SelectedMetricGroups`: The metric groups you wish to send. Enter `All` if you want all available metrics. Otherwise, list the names of desired metric groups, spelled exactly as they are below, separated by single spaces. See [Metrics collected by this integration](#metric-groups-collected-by-this-integration) for options. 
```

Comment:
"below" seems a little vague here. Maybe move the link here? So ..exactly as they are in [metrics collected].... Then you can delete the last sentence.